cloudalerts/v2/loggers/structlog.py

```diff
@@ -59,12 +59,6 @@ def decode_json_then_enqueue(
     _Worker.enqueue = decode_json_then_enqueue
 
 
-def event_uppercase(logger, method_name, event_dict):
```

Comment:
Just need to remove the associated tests and it's good to merge!

Reply:
yup...on it.

Reply:
done.